.bluemix/pipeline.yml

```diff
@@ -26,9 +26,9 @@ stages:
       application: ${CF_APP}
     script: |-
       #!/bin/bash
-      cf create-service watson_vision_combined free visual-recognition-wws
+      cf create-service watson_vision_combined lite visual-recognition-wws
       pushd server
-      ./custom-model.sh
+      # ./custom-model.sh
```

Comment:
this is commented out, if it's not needed then delete the line and the script, otherwise fix `custom_model.sh` -- what error was/is it creating?

Reply:
We decided to remove the extra script since there's too many components to maintain. Plus the server will submit the model training job if the model is not exist. Therefore, we technically don't need any instructions for creating custom models(plus the UI and API is constantly changing).